### ISSUE TITLE
Two-Weapon Fighting Action remove Ability Modifier from damage

### ISSUE THREAD

**Issue by LogosWorks** [ZerdBG3/DnD-Rebalancing#7]
Currently (hopefully a bug) off hand attack adds Ability Modifier to damage.
This should only happen with _Two-Weapon Fighting_ Fighting Style.
### Two-Weapon Fighting
PHB p195

When you take the Attack action and attack with a light melee weapon that you're holding in one hand, you can use a bonus action to attack with a different light melee weapon that you're holding in the other hand. You don't add your ability modifier to the damage of the bonus attack, unless that modifier is negative.If either weapon has the thrown property, you can throw the weapon, instead of making a melee attack with it.

**Comment by LogosWorks** [comment]
Worth looking at /Public/Shared/Stats/Generated/Data/Data.txt:
ln 453 and 455
```
key "DualWieldingPlayersDefaultOn","1"

key "DualWieldingNPCsDefaultOn","1"
```

**Comment by FelipeRenault** [comment]
Yes, this is a known bug and I've reported it. Looking at the text files from the target skills from Main Hand and Offhand attack, I've found nothing that defines how the attack and damage is rolled. I'm assuming this is something in their code (thus it's hardcoded).
The entries you've found on extra data are simply the default values for automatic dual wielding, which can be toggled by pressing `V`. It's only to determine if you're automatically spending your bonus action to take an offhand attack when you use your attack
